CONTRIBUTING.md

```diff
@@ -91,13 +91,12 @@ Here's how to run the test suite:
 
 ## Vendoring
 
-We keep a cached copy of dependencies within the `vendor/` directory,
-managing updates via [Modules](https://golang.org/cmd/go/#hdr-Module_maintenance).
+We managing updates via [Modules](https://golang.org/cmd/go/#hdr-Module_maintenance).
 
-Pull requests should only include `vendor/` updates if they are part of
+Pull requests should only include `go.mod`, `go.sum` updates if they are part of
 the same change, be it a bugfix or a feature addition.
 
-The `vendor/` update needs to be justified as part of the PR description,
+The `go.mod`, `go.sum` update needs to be justified as part of the PR description,
 and must be verified by the reviewers and/or merger to always reference
 an existing upstream commit.
```

Comment:
If we use Go Modules, we do not need to check the upstream commit again. This check was required by the committed `vendor` directory in my mind.

Reply:
The intent of this paragraph is to require a justification whenever a dependency is added or modified. I think the proposed change is correct. Adding a new dependency previously meant modifying **vendor/, go.mod and go.sum**. Requiring an explanation when **vendor/** was modified implied requiring an explanation when **go.mod and go.sum** was modified. Now that **vendor/** is removed, there is a need to rephrase the requirement.